lib/components/navbar.vue

```diff
@@ -33,7 +36,7 @@
             },
             toggleBreakpoint: {
                 type: String,
-                default: 'sm'
```

Comment:
What if `toggleable` is `true` and user not providing `toggleBreakpoint` which is null by default ? :))